File: objc/Project12/Project12/SharedCode.m
//
//  SharedCode.m
//  Project12
//
//  Created by Jinwoo Kim on 2/3/21.
//

#import "SharedCode.h"

NSURL * getDocumentsDirectory(void) {
    NSArray<NSURL *> *paths = [NSFileManager.defaultManager URLsForDirectory:NSDocumentDirectory inDomains:NSUserDomainMask];
    return paths[0];
}

File: swift/Project12/Project12/SharedCode.swift
//
//  SharedCode.swift
//  Project12
//
//  Created by Jinwoo Kim on 2/2/21.
//

import Foundation

func getDocumentsDirectory() -> URL {
    let paths = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)
    return paths[0]
}
